cli/setup-owl.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+# Copyright (c) 2021, General Electric Company and Galois, Inc.
+
+set -e
+
+RACK_DIR=$(realpath "$(dirname "$0")"/..)
+DOCKER_TAG="v9.0"
+MODE=copy
+
+usage() {
+        echo "Usage: setup-owl.sh [-h] [-b] [-t TAG]"
+        echo "   -h    print help"
+        echo "   -b    build OWL files using Web SADL"
```

Comment:
Web SADL is a different thing (Typefox's theiasadl).  The Docker image is SADL-Eclipse or sadl-eclipse.